src/DynamoPackages/PackageLoader.cs

```diff
@@ -643,16 +644,8 @@ private Version CheckAndGetPackageVersion(string version, string name, string di
         /// </summary>
         /// <param name="packageDirectoryPath">path to package location</param>
         /// <param name="discoveredPkg">package object to check</param>
-        private void CheckPackageNodeLibraryCertificates(string packageDirectoryPath, Package discoveredPkg)
+        private static void CheckPackageNodeLibraryCertificates(string packageDirectoryPath, Package discoveredPkg)
         {
-            var dllfiles = new System.IO.DirectoryInfo(discoveredPkg.BinaryDirectory).EnumerateFiles("*.dll");
-            if (discoveredPkg.Header.node_libraries.Count() == 0 && dllfiles.Count() != 0)
-            {
-                Log(String.Format(
-                    String.Format(Resources.InvalidPackageNoNodeLibrariesDefinedInPackageJson,
-                    discoveredPkg.Name, discoveredPkg.RootDirectory)));
-            }
```

Comment:
Remove the node library dll check because Built-In package moving forward may not always include nodes so this is no longer needed, even the logging